content/docs/gto/get-started.md

```diff
@@ -0,0 +1,170 @@
+$ gto show
+╒══════════╤══════════╤════════╤═════════╤════════════╕
+│ name     │ latest   │ #dev   │ #prod   │ #staging   │
+╞══════════╪══════════╪════════╪═════════╪════════════╡
+│ churn    │ v3.1.1   │ v3.1.1 │ v3.0.0  │ v3.1.0     │
+│ segment  │ v0.4.1   │ v0.4.1 │ -       │ -          │
+│ cv-class │ v0.1.13  │ -      │ -       │ -          │
+╘══════════╧══════════╧════════╧═════════╧════════════╛
+```
+
+Here we have 3 models: `churn`, `segment` and `cv-class`. The latest versions of
+them are shown in the column named `latest`. The latest is selected as the one
+having the greatest [SemVer](https://semver.org).
+
+Model versions could be promoted to different stages. Here we have 3 of them:
+`dev`, `prod` and `staging`. When a model was never promoted to a stage, we see
+`-` in the field.
+
+## Registering versions and assigning stages
+
+GTO can [register version](/doc/gto/command-reference/register) of artifacts and
+[assign stages to them](/doc/gto/command-reference/assign). Both functionalities
+work in a similar way, so let's walkthough only one of them here.
+
+Let's assume the version `v0.1.13` of `cv-class` looks very promising, and now
+we want to promote it to `dev` to test it:
+
+```cli
+$ gto assign cv-class --version v0.1.13 --stage dev
+Created git tag 'cv-class#dev#1' that assigns stage to version 'v0.1.13'
+To push the changes upstream, run:
+    git push origin cv-class#dev#1
+```
+
+GTO created a Git tag with a special format that contains instruction to assign
+a stage to a version. We can push to Git repository to start the CI, but let's
+ensure that changed our Registry first.
+
+```cli
+$ gto show
+╒══════════╤══════════╤═════════╤═════════╤════════════╕
+│ name     │ latest   │ #dev    │ #prod   │ #staging   │
+╞══════════╪══════════╪═════════╪═════════╪════════════╡
+│ churn    │ v3.1.1   │ v3.1.1  │ v3.0.0  │ v3.1.1     │
+│ segment  │ v0.4.1   │ v0.4.1  │ -       │ -          │
+│ cv-class │ v0.1.13  │ v0.1.13 │ -       │ -          │
+│ awesome  │ v0.0.1   │ -       │ -       │ -          │
+╘══════════╧══════════╧═════════╧═════════╧════════════╛
+```
+
+The `gto show` output confirms our expectation.
+
+## Acting downstream
+
+The power of using Git tags to register versions and assign stages is simple: we
+can act upon them in well-known way - in CI/CD.
+
+<details>
+
+### Click to set up a working repository to try this.
+
+Let's fork the [example-gto repo](https://github.com/iterative/example-gto) repo
+(you'll need a [GitHub](https://github.com/signup) account first). For CI/CD to
+start, you'll need to enable it on the "Actions" page of your fork.
+
+1. [Fork the repo](https://github.com/iterative/example-gto/fork). Make sure you
+   uncheck "Copy the `main` branch only" to copy Git tags as well:
+   <img width="877" alt="image" src="https://user-images.githubusercontent.com/6797716/199275275-439335f4-6f54-4cd7-910d-fc29ad3c095c.png">
+
+2. Enable workflows in your repo, for a Git tag to trigger CI:
```

Comment:
```suggestion
2. Enable workflows in your repo for a new Git tag to trigger CI:
```

Is that the default behavior or does it imply more specific GH Action config?

Reply:
It's the default.